src/cpp/multisolver_interface/SolverCplex.h

```diff
@@ -74,9 +74,12 @@ class SolverCplex : public SolverAbstract {
  public:
   virtual void write_prob_mps(const std::filesystem::path &filename) override;
   virtual void write_prob_lp(const std::filesystem::path &filename) override;
+  virtual void write_basis(const std::filesystem::path &filename) override{};
 
   virtual void read_prob_mps(const std::filesystem::path &filename) override;
-  virtual void read_prob_lp(const std::string &filename) override;
+  virtual void read_prob_lp(const std::filesystem::path &filename) override;
```

Comment:
i think `virtual` keyword is not needed here.

Reply:
Actually, all `virtual` keywords in files from the child classes of `SolverAbstract` are not needed. I keep it here to be coherent with the rest of the file, but I can remove it everywhere if you think it is worth it.

Reply:
Your are right and the pr is already large, let's do that in dedicated branch.